### PR TITLE
Feat/add handling missing env

### PR DESCRIPTION
**Pull request by AndriiBogomolov**
1. Added handling missing PG_CONNECTION_STRING
2. Added handling missing CREDENTIALS_PATH
3. Updated README.md

### CODE REVIEW

**Review by djeck1432 — APPROVED** [review]
Looks good to me

File: data_handler/handlers/helpers.py [hunk 1]
```diff
@@ -18,7 +19,7 @@
 
 GS_BUCKET_NAME = "derisk-persistent-state"
 ERROR_LOGS = set()
-
+logger = logging.getLogger(__name__)
```
Comment (lukaspetrasek) [inline data_handler/handlers/helpers.py:22]:
```suggestion


logger = logging.getLogger(__name__)

```

File: README.md [hunk 2]
```diff
@@ -6,10 +6,6 @@ Monorepo with components required for the implementation of DeRisk on Starknet.
 
 See [our contributor guidelines](https://github.com/CarmineOptions/derisk-research/blob/master/CONTRIBUTING.md).
 
-## Set env variables
```
Comment (lukaspetrasek) [inline README.md:9]: This is still relevant, no?

**Comment by AndriiBogomolov** [comment]
No, it's not more relevant

File: data_handler/handlers/helpers.py (hunk 1 above)
Comment (lukaspetrasek) [reply data_handler/handlers/helpers.py:22]: @AndriiBogomolov Pls address this comment.

**Pull request merged**